docs/contributing/index.rst

```diff
@@ -89,19 +94,132 @@ versions are maintained through the community. Divio provides long term support
 `commercial support <http://divio.ch/en/commercial-support/>`_.
 
 
-*************
+.. _branch_policy:
+
 Branch policy
-*************
+=============
+
+.. versionchanged:: 3.3
```

Comment:
As a reader, I don't think these annotations provide much value unless you describe what changed. (ala https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features)